pegasus/helpers/section_api_helpers.rb

```diff
@@ -266,6 +292,10 @@ def self.valid_script_id?(script_id)
     valid_scripts.find {|script| script[:id] == script_id.to_i}
   end
 
+  def self.valid_course_id?(course_id)
+    valid_courses.find {|course| course[:id] == course_id.to_i}
```

Comment:
Unless documented otherwise, I'd expect `course_id` to be an integer. Why the cast via `to_i`?